pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -80,6 +81,7 @@ const (
 	argMemBalloon10period = `<memballoon model="virtio-non-transitional" freePageReporting="on">
       <stats period="10"></stats>
     </memballoon>`
+	BlockPVCName = "pvc_block_test"
```

Comment:
unexport this please `s/BlockPVCName/blockPVCName/`

Reply:
Nice catch, ack